### PR TITLE
IPX-529

### PR DESCRIPTION
**Pull request by TheBoyzz**
Sakura Momo 😁

### CODE REVIEW

**Comment by TheBoyzz** [comment]
Nice test

**Review by TheBoyzz — COMMENTED** [review]
- **Goodcode**

File: deku.css [hunk 1]
```diff
@@ -1,4 +1,8 @@
 deku {
     background-color: red;
```
Comment (TheBoyzz) [inline deku.css:2]: Ohhhhhhhhh

File: deku.css [hunk 2]
```diff
@@ -1,4 +1,8 @@
...
+    text-align: center;
```
Comment (TheBoyzz) [inline deku.css:3]: That WTF

File: deku.css [hunk 3]
```diff
@@ -1,4 +1,8 @@
...
+    display: flex;
+    color: blue;
     width: 100%;
```
Comment (TheBoyzz) [inline deku.css:6]: T_T

**Review by TheBoyzz — COMMENTED** [review]
Alrighttt

File: deku.css (hunk 2 above)
Comment (BlueBoyzz) [reply deku.css:3]: 56156156156

File: deku.css [hunk 4]
```diff
@@ -1,4 +1,8 @@
 deku {
     background-color: red;
+    text-align: center;
+    display: flex;
+    color: blue;
```
Comment (BlueBoyzz) [inline deku.css:5]: Holyy Crap

**Review by BlueBoyzz — COMMENTED** [review]
After Fix

**Pull request merged**